.github/labels.yml

```
@@ -86,6 +86,30 @@
   description: "Work on Test Runners"
   color: "ffffff"
 
+# The `x:rep/<value>` labels describe the amount of reputation to award
+# 
```

```suggestion
#
```

Consider not merging for now. We could consider recreating this PR, which is not modifiable.

See https://github.com/exercism/org-wide-files/pull/204

---

Oh, well caught!